sui/src/sui_node.rs

```diff
@@ -80,20 +75,15 @@ impl RpcGatewayServer for SuiNode {
         todo!()
     }
 
-    async fn get_object_info(&self, _object_id: ObjectID) -> RpcResult<ObjectRead> {
-        todo!()
-    }
-
-    async fn get_object_typed_info(
-        &self,
-        _object_id: ObjectID,
-    ) -> RpcResult<GetObjectInfoResponse> {
+    async fn get_object_info(&self, _object_id: ObjectID) -> RpcResult<SuiObjectRead> {
```

Comment:
```suggestion
    async fn get_object_info(&self, _object_id: ObjectID) -> RpcResult<GetObjectInfoResponse> {
```
nit: I would prefer to name this `GetObjectInfoResponse` to signify this only used in the RPC layer and is more intuitive to understand the purpose of this struct